e3sm_diags/driver/tropical_subseasonal_driver.py

```diff
@@ -220,6 +207,13 @@ def wf_analysis(x, **kwargs):
     # OPTIONAL kwargs:
     # segsize, noverlap, spd, latitude_bounds (tuple: (south, north)), dosymmetries, rmvLowFrq
 
+    # Interpolate missing values along longitude before spectral analysis
+    if np.any(np.isnan(x)):
+        logger.info(
+            "Interpolating missing values along longitude before spectral analysis"
+        )
+        x = x.interpolate_na(dim="lon", method="linear", fill_value="extrapolate")
+
```

Comment:
A simple linear interpolation to fill in missing for U850

Reply:
Will this linear interpolation unintentionally affect other variables, or do we want other variables with missing values to be handled too?